src/test/resources/org/broadinstitute/hellbender/tools/walkers/varianteval/VariantEval/expected/testAlleleCountStrat.expected.txt

```diff
@@ -0,0 +1,12 @@
+#:GATKReport.v1.1:1
```

Comment:
maybe i'm missing something, but that path is dictated largely by class name?  as you noted, VariantEval is this sprawling tool and has its own package.  Most tools dont.  Since this path is based off the tool package (org.broadinstitute.hellbender.tools.walkers.varianteval.VariantEval), that's where redundancy comes from.  I could see argument about refactoring the location of the tool, but it seems to me that the data should mirror that?

I supposed I could override various test methods to change the expected file location, but this doesnt seem like a great idea.

Reply:
Oh yeah, forgot about that. Leave it as is then.